consumption.tex

```diff
@@ -2,57 +2,44 @@ \section{Consumption}
-
-We wish to allow users to exchange a legacy cryptocurrency for a new cryptocurrency. We call the former the \emph{source cryptocurrency}, and its blockchain the \emph{source blockchain}. We call the latter the \emph{target cryptocurrency} and its corresponding blockchain the \emph{target blockchain}. The target blockchain may support burning from multiple source blockchains.
-
-We now describe how a smart contract on the target blockchain can verify that a burn has taken place on the source blockchain. Note how this puts a constraint on the target blockchain to have smart contract capabilities. However there is no such constraint for the source blockchain. Such a mechanism allows a target cryptocurrency to bootstrap through burns on another source blockchain as outlined in Section~\ref{section:introduction}.
+This chicken-and-egg situation presents the need for a solution that circumvents exchanges and allows users to acquire the cryptocurrency in a usable manner, for it to gain traction. We propose utilizing proof-of-burn to allow users to obtain capital on a new cryptocurrency, automatically and without trusted third parties, by burning a legacy cryptocurrency. We call the former the \emph{source cryptocurrency}, and its blockchain the \emph{source blockchain}. We call the latter the \emph{target cryptocurrency} and its corresponding blockchain the \emph{target blockchain}. The target blockchain may support burning from multiple source blockchains.
 
 \noindent
 \textbf{Workflow.}
-A user who wishes to acquire a target cryptocurrency first forms a burn address valid in the source blockchain which encodes her receiving address on the target blockchain by using it as a tag. She then sends an amount of the source cryptocurrency to that address. She submits a proof of this burn to a smart contract on the target blockchain, where it is verified and she is credited an equivalent amount of target cryptocurrency on her receiving address. Proof-of-burn verification happens in either a centralized manner which is lighter on computation but requires a federation, or in a decentralized manner using Non-Interactive Proofs of Proof-of-Work (NIPoPoWs).
+A user who wishes to acquire a target cryptocurrency first forms a burn address valid in the source blockchain which encodes her receiving address on the target blockchain by using it as a tag. She then sends an amount of the source cryptocurrency to that address. She submits a proof of this burn to a smart contract on the target blockchain, where it is verified and she is credited an equivalent amount of target cryptocurrency on her receiving address. Proof-of-burn verification happens in either a centralized manner which is lighter on computation, or in a decentralized manner using NIPoPoWs.
```

Comment:
of the source cryptocurrency -> of source cryptocurrency

Reply:
Remove " and can be decentralized by mak- ing use of Non-Interactive Proofs of Proof-of-Work (NIPoPoWs) [12]." from intro, spell our Non-Interactive Proofs of Proof-of-Work and citation here. Cite KLS also.